README.md

```diff
@@ -13,6 +13,10 @@ Run npm start and waits to url before testing
 
 Url that npm start command waits for
 
+### `pr_author_username`
```

Comment:
Se esse input for obrigatório, seria interessante atualizar em como consumir a action com esse input, nas seções [`Usage example`](https://github.com/betrybe/jest-evaluator-action#usage-example) e [`How to get result output`](https://github.com/betrybe/jest-evaluator-action#how-to-get-result-output), não?